reference/forms/types/week.rst

```diff
@@ -0,0 +1,187 @@
+.. index::
+   single: Forms; Fields; WeekType
+
+WeekType Field
+==================
+
+.. versionadded:: 4.4
+
+    The ``WeekType`` Type was introduced in Symfony 4.4.
+
+This field type allows the user to modify data that represents a specific
+ISO 8601 week number  (e.g. ``1984-W05``).
```

Comment:
Please make ISO 8601 a link to:
https://en.m.wikipedia.org/wiki/ISO_8601

Please remove one whitespace before „(„